ls1intum/Artemis
# `Development`: Unify file endpoints

### Checklist
#### General


#### Server

### Motivation and Context


This follows up #5574 and covers all endpoints that use file tokens for authentication.

### Description

I added a new annotation, replaced the occurrences and added the annotation to the documentation.
There are no changes to the mapping as all endpoints in question already have the path prefix `api/files/`

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student

1. Upload a file somewhere (e.g. lecture)
2. Try to download the file as a student
3. All should work fine as before

### Review Progress





#### Code Review
#### Manual Tests
